src/Modules/SaveTimesheet.js

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+import moment from 'moment';
+
+const saveTime = async (begin, end) => {
+	const apiUrl = "https://demo-stable.kimai.org/api/";
+	const proxyUrl = "https://cors-anywhere.herokuapp.com/";
```

Comment:
In other features I have worked on, we dont need to use the proxyUrl. Can you try to remove it again and see if it still works?

Reply:
Error 403: Forbidden and Network error received when doing so

Reply:
i also think we need it since making post requests from localhost seems to a big "nono"